turbopuffer/namespace.py

```diff
@@ -63,6 +133,24 @@ def refresh_metadata(self):
             }
         else:
             raise APIError(response.status_code, 'Unexpected status code', response.get('content'))
+    
+    def schema(self) -> NamespaceSchema:
+        """
+        Returns the current schema for the namespace.
+        """
+        response = self.backend.make_api_request('vectors', self.name, 'schema', method='GET')
+        return parse_namespace_schema(response["content"])
+
+    def update_schema(self, schema_updates: NamespaceSchema):
+        """
+        Writes updates to the schema for a namespace.
+        Returns the final schema after updates are done.
+
+        See https://turbopuffer.com/docs/schema for specifics on allowed updates.
+        """
+        request_payload = json.dumps({key: value.as_dict() for key, value in schema_updates.items()}).encode()
+        response = self.backend.make_api_request('vectors', self.name, 'schema', method='POST', payload=request_payload)
+        return parse_namespace_schema(response["content"])
```

Comment:
nit, kind of seems like these should be lower in the file, at least after exists()